test/test_onnx.py

```diff
@@ -285,8 +285,6 @@ def get_test_images(self):
         test_images = [image2]
         return images, test_images
 
-    @unittest.skip("Disable test until Resize opset 11 is implemented in ONNX Runtime")
-    @unittest.skipIf(torch.__version__ < "1.4.", "Disable test if torch version is less than 1.4")
     def test_faster_rcnn(self):
         images, test_images = self.get_test_images()
```

Comment:
I think we might also need to change the `min_size` and `max_size` for this test to be changed.
So something like
```python
model.transform.min_size = 300
model.transform.max_size = 300
```